### PR TITLE
Matches crawler

### PR DESCRIPTION
**Pull request by tyronedamasceno**
Traz todos os jogos do brasileirão séries A e B, Copa do Nordeste e Copa do Brasil

### CODE REVIEW

File: crawler_copa_brasil.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    for box in boxes:
+        info = box.find('div', {'class':'placar-jogo-informacoes'}).text
+
+        mandante = box.find('span', {'class':'placar-jogo-equipes-item placar-jogo-equipes-mandante'})
+        mandante_sigla = mandante.find('span', {'class':'placar-jogo-equipes-sigla'}).text
+        mandante = mandante.find('span', {'class':'placar-jogo-equipes-nome'}).text
+
+        visitante = box.find('span', {'class':'placar-jogo-equipes-item placar-jogo-equipes-visitante'})
+        visitante_sigla = visitante.find('span', {'class':'placar-jogo-equipes-sigla'}).text
+        visitante = visitante.find('span', {'class':'placar-jogo-equipes-nome'}).text
```
Comment (ivancrneto) [inline crawler_copa_brasil.py]: Could you please use English variables here?

File: crawler_copa_brasil.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        
+        output = '{} x {}\n{}\n'.format(mandante, visitante, info)
+        print(output)
```
Comment (ivancrneto) [inline crawler_copa_brasil.py:19]: What about logging instead of print?

File: crawler_copa_brasil.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+ff = webdriver.Firefox()
+ff.get(url_copa_brasil)
+bs_obj = bs(ff.page_source, 'html.parser')
+
+seta_fase_esq = ff.find_element_by_class_name('gui-icon-arrow-left-highlight')
+seta_fase_dir = ff.find_element_by_class_name('gui-icon-arrow-right-highlight')
+
+for i in range(8):
+    seta_fase_esq.click()
```
Comment (ivancrneto) [inline crawler_copa_brasil.py]: Same for the 6 lines above please, English vars 😺

File: requirements-dev.txt [hunk 4]
```diff
@@ -1,2 +1,34 @@
...
+Pygments==2.2.0
+python-dateutil==2.7.5
+requests==2.19.1
+rsa==4.0
+selenium==3.141.0
+six==1.11.0
+traitlets==4.3.2
+uritemplate==3.0.0
+urllib3==1.23
+wcwidth==0.1.7
```
Comment (ivancrneto) [inline requirements-dev.txt:34]: looks like you put production requirements in the dev file, can we have that fixed please?

File: crawler_serie_a_b.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        visitante = visitante.find('span', {'class':'equipes__nome'}).text
+        
+        info = box.find('div', {'class':'jogo__informacoes'})
+        data = info.find('span', {'class':'jogo__informacoes--data'}).text.strip()
+        local = info.find('span', {'class':'jogo__informacoes--local'}).text.strip()
+        hora = info.find('span', {'class':'jogo__informacoes--hora'}).text.strip()
+
+        print('\n{} x {}\n{} {} {}\n'.format(mandante, visitante, data, hora, local))
+
+    seta_dir.click()
```
Comment (ivancrneto) [inline crawler_serie_a_b.py]: Looks like the two files above have some stuff in common with the first one. Can we do something to reuse the code?

File: crawler_copa_ne.py [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+for i in range(3):
+    seta_fase_dir.click()
+
+    bs_obj = bs(ff.page_source, 'html.parser')
+    fase = bs_obj.find('span', {'class':'tabela-navegacao-seletor-ativo'}).text
+    print(fase, "\n")
+    boxes = bs_obj.find_all('div', {'class':'placar-jogo'})
+
+    print_matches(boxes)
+
```
Comment (ivancrneto) [inline crawler_copa_ne.py:83]: English here in this file also 😅

**Review by ivancrneto — CHANGES_REQUESTED** [review]

File: crawler_copa_brasil.py (hunk 2 above)
Comment (tyronedamasceno) [reply crawler_copa_brasil.py:19]: What is How to do this?

File: crawler_copa_brasil.py (hunk 1 above)
Comment (tyronedamasceno) [reply crawler_copa_brasil.py]: Of course!

File: crawler_copa_brasil.py [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    sleep(1)
+
+for i in range(8):
+    sleep(1)
+    bs_obj = bs(ff.page_source, 'html.parser')
+    fase = bs_obj.find('span', {'class':'tabela-navegacao-seletor-ativo'}).text
```
[the remark this answers is not in the document]
Comment (tyronedamasceno) [reply crawler_copa_brasil.py]: I didn't understand

File: crawler_copa_brasil.py [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+    print(fase, "\n")
+
+    boxes = bs_obj.find_all('div', {'class':'placar-jogo'})
+
+    print_matches(boxes)
+
+    seta_fase_dir.click()
```
[the remark this answers is not in the document]
Comment (tyronedamasceno) [reply crawler_copa_brasil.py]: I didn't understand... again kkk

File: requirements-dev.txt (hunk 4 above)
Comment (tyronedamasceno) [reply requirements-dev.txt:34]: Ok, i'll work on it, got confused

File: crawler_serie_a_b.py (hunk 5 above)
Comment (tyronedamasceno) [reply crawler_serie_a_b.py]: Sure. I'll think about and fix it

File: crawler_copa_brasil.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (tyronedamasceno) [reply crawler_copa_brasil.py]: Actually, i don't know the meme

File: crawler_copa_brasil.py (hunk 2 above)
Comment (ivancrneto) [reply crawler_copa_brasil.py:19]:
``` python
logger = logging.getLogger(__name__)
logger.info(output)
```

File: crawler_copa_brasil.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (ivancrneto) [reply crawler_copa_brasil.py]: it's from a movie, Pulp Fiction, "English Motherfucker"

File: crawler_copa_brasil.py (hunk 8 above)
[the remark this answers is not in the document]
Comment (ivancrneto) [reply crawler_copa_brasil.py]: https://www.youtube.com/watch?v=AJSas8nwnnU

File: crawler_copa_brasil.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (tyronedamasceno) [reply crawler_copa_brasil.py]: Aaaah, okay kkk

**Pull request closed without merging**